ci-operator/step-registry/syndesisio/setup-syndesis-qe-env/syndesisio-setup-syndesis-qe-env-commands.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+oc project default || true
+oc registry login || true
+oc create --as system:admin user kubeadmin || true
```

Comment:
Isn't the `kubeadmin` already created

Reply:
Yes, but it is not found by the test suite in the default namespace unless these commands are run.